deployment/desktop/software-appdata.md

```diff
@@ -0,0 +1,137 @@
+   applications typically use a reverse-DNS style name, for example
+   *org.example.Frobnicator.desktop*.
+
+1. Create a PNG icon for the file at least 64x64 in size, with a
+   transparent background.  The icon should be named as in the *Icon*
+   field above, for example *frob.png*.  Your package should install
+   this file in one of the following locations:
+   - */usr/share/icons*
+   - */usr/share/icons/hicolor/apps*
+   - */usr/share/${app_name}/icons*
+   
+1. Verify the desktop file with the *desktop-file-validate* command:
+
+   ```console
+$ desktop-file-validate org.example.Frobnicator.desktop
+   ```
+   
+## Create an AppData file
+
+The AppData file provides XML metadata that follows the [AppStream
+specification](https://www.freedesktop.org/software/appstream/docs/),
+and is used by the GNOME Software app to display the app to users in
+the software center.
+
+1. Create an XML file for your application that follows the template
+   below. Note that you should consult the [AppStream
+   specification](https://www.freedesktop.org/software/appstream/docs/)
+   to provide correct data for your application in this file.  Here is
+   an example for the Frobnicator app:
+   
+   ```console
+    <?xml version="1.0" encoding="UTF-8"?>
+    <component type="desktop">
+      <id>org.frobnicator.Frobnicator.desktop</id>
+      <metadata_license>CC0-1.0</metadata_license>
+      <project_license>GPL-2.0+</project_license>
+      <name>Frobnicator</name>
+      <summary>Frobnicate a PNG into a 3D object</summary>
+      <description>
+        <p>You can use Frobnicator to turn a PNG file into a 3D object magically, if you have the appropriate Frob interface card installed on your system.</p>
+      </description>
+      <screenshots>
+        <screenshot type="default">
+          <image>https://frobnicator.org/git/browse/frobnicator/plain/data/appdata/frob-startup.png</image>
+          <caption>Initial screen for the application</caption>
+        </screenshot>
+      </screenshots>
+      <releases>
+        <release version="1.0.0" date="2017-12-02">
+          <description>
+            <p>This is the first stable release</p>
+          </description>
+        </release>
+      </releases>
+      <url type="homepage">https://example.org/Frobnicator</url>
+      <provides>
+        <binary>frob</binary>
+      </provides>
+      <mimetypes>
+        <mimetype>application/x-frob</mimetype>
+      </mimetypes>
+    </component>
+	```
+
+1. Next, make the appropriate 16:9 aspect ratio screenshots for your
+   package, showing the application in action.  To make screenshot
+   creation easier, use the GNOME Shell extension called *Screenshot
+   Window Sizer* which you installed earlier.  To resize the window of
+   your application to 16:9 format, focus it and press *Ctrl+Alt+s*.
+   Your application window is resized to a perfect 16:9 aspect ratio
+   so you can screenshot it with *Alt+PrtScn*.
```

Comment:
Likewise I do not think this is a place for a basic usage of any `Screenshot` tool. You can link to a relevant guide.